aioitertools/itertools.py

```diff
@@ -247,9 +254,25 @@ def pred(x):
                 yield item
 
 
+# pylint: disable=unused-argument,missing-docstring,undefined-variable
+# pylint: disable=function-redefined,multiple-statements
```

Comment:
Some of these would be reasonable to add to the global ignore list in `.pylint`, like unused-argument, missing-docstring, and function-redefined.  Would also help cut down on the linter directives.